site/en/docs/multidevice/user-agent/index.md

```diff
@@ -122,8 +183,7 @@ Version/4.0 <mark>Chrome/43.0.2357.65</mark> Mobile Safari/537.36</code></pre>
 
 **WebView UA in Android 10 and above**
 
-In the newer versions of WebView, you can differentiate the WebView by looking for the `wv` field as
-highlighted in the following string.
+In the newer versions of WebView, you can differentiate the WebView by looking for the `wv` field as highlighted in the following string.
```

Comment:
Note that we added support for User-Agent Client Hints to WebView in M119. So a site can look for "Android WebView" in the brandlist of the Sec-CH-UA hint as well (which is sent by default):

Sec-CH-UA: "Android WebView";v="113", "Chromium";v="113", "Not-A.Brand";v="24"